viewing_party/party.py

```diff
@@ -1,23 +1,166 @@
 # ------------- WAVE 1 --------------------
-
 def create_movie(title, genre, rating):
-    pass
+    if title != None and genre != None and rating != None:

```

Comment:
Nice! We can change this to `if title is not None and genre is not None...`. (see my explanation for using `is` above.)